open-api/lang-parser/src/main/java/io/deephaven/lang/generated/ChunkerMixin.java

```diff
@@ -648,14 +747,62 @@ default boolean isTypedInvoke(boolean ctor) {
             .matches(true);
     }
 
+    default boolean isMethodDecl() {
+        final PeekStream matcher = peek();
+        matcher.whitespace()
+            .withTokens(
+                    // blech. this is ...ugly AF. We might need to delete this, and make actual productions for GroovyMethodDecl
```

Comment:
mark as todo? at least clean up the comment a bit?

Reply:
deleted the whole method. it was a failed experiment.